Makefile

```diff
@@ -3,6 +3,9 @@ IMG ?= fluxcd/image-reflector-controller
 # Produce CRDs that work back to Kubernetes 1.16
 CRD_OPTIONS ?= crd:crdVersions=v1
 
+ENVTEST_BIN_VERSION?=1.22.0
+KUBEBUILDER_ASSETS?=$(shell $(SETUP_ENVTEST) use -i $(ENVTEST_BIN_VERSION) -p path)
```

Comment:
I couldn't remember if `?=` expanded once or at every mention, so I looked it up: https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html#SEC59 (answer: yes, it's expanded at every mention). This is usually a bit dangerous. On the other hand, it wouldn't work to use a simple expansion (`:=`) because it relies on SETUP_ENVTEST, which is only set in the rule for `setup_envtest`. I can't think of a nicer way to do this though, can you?

Reply:
No, I've only seen this pattern everywhere. I'm not aware of anything nicer.